frontend/src/components/Browse.tsx

```diff
@@ -19,84 +18,68 @@ interface Props {
   }
 
   async componentDidMount() {
     try {
       const res = await axios.get("http://localhost:8080/api/rooms");
-      const numrooms = Object.keys(res.data).length;
-      let numParsed = 0;
-      for (const key in res.data) {
-        getYoutubeTitle(res.data[key].currVideoId, (err: any, title: any) => {
-          this.state.vidTitle[res.data[key].currVideoId] = title;
-          numParsed++;
-          if (numParsed === numrooms) {
-            this.setState({
-              roomList: res.data
-            });
-          }
-        });
-      }
+      this.setState({
+        roomList: res.data
+      });
+      window.addEventListener("resize", () => {
+        this.setState({ width: window.innerWidth, height: window.innerHeight });
+      });
     } catch (err) {
       console.log(err);
     }
   }
 
-  renderRoom(room: any, key: any) {
+  mediaCard(room: any, key: any) {
     console.log(this.state);
+    const { classes } = this.props;
     const roomId = key.split(":")[1];
     return (
       <Grow in={true} timeout="auto" key={key}>
-        <TableRow hover key={key}>
-          <TableCell>
-            <img
-              alt={key}
-              src={`https://img.youtube.com/vi/${room.currVideoId}/default.jpg`}
-              onClick={event => (window.location.href = "/rooms/" + roomId)}
-            ></img>
-          </TableCell>
-          <TableCell>{this.state["vidTitle"][room.currVideoId]}</TableCell>
-          <TableCell>{room.name}</TableCell>
-          <TableCell>{roomId}</TableCell>
-          <TableCell>
-            <Button onClick={event => (window.location.href = "/rooms/" + roomId)}>Go to Room</Button>
-          </TableCell>
-        </TableRow>
+        <GridListTile
+          className="tile"
+          key={`https://img.youtube.com/vi/${room.currVideoId}/default.jpg`}
+          onClick={event => (window.location.href = "/rooms/" + roomId)}
+        >
+          <img src={`https://img.youtube.com/vi/${room.currVideoId}/default.jpg`} alt={room.name} />
+          <GridListTileBar title={room.name} subtitle={<span>Room ID: {roomId}</span>} />
+        </GridListTile>
       </Grow>
     );
   }
 
   render() {
     return (
-      <div className="roomList">
-        <Paper>
-          <Table>
-            <TableHead>
-              <TableRow>
-                <TableCell>Video Playing</TableCell>
-                <TableCell>Video Name</TableCell>
-                <TableCell>Room Name</TableCell>
-                <TableCell>Room Key</TableCell>
-                <TableCell></TableCell>
-              </TableRow>
-            </TableHead>
-            <TableBody>
-              {Object.keys(this.state.roomList).map(key => {
-                return this.renderRoom(this.state.roomList[key], key);
-              })}
-            </TableBody>
-          </Table>
-        </Paper>
+      <div className="browse">
+        <div className="title">
+          <img className="logo" src={logo}></img>
```

Comment:
I think it would be good UX to have the sync logo bring you back to the home page on click. We should add that for all pages with a logo.